mlops/Experiment.py

```diff
@@ -66,7 +66,7 @@ def check_gpu(self):
             request_gpu = True
 
         logger.info(f'GPU requested: {request_gpu}, cuda_available {torch.cuda.is_available()}')
-        if torch.cuda.is_available() and request_gpu:
```

Comment:
Do we definitely want to delete `torch.cuda.is_available()` here?

Reply:
My thinking is at this point we're essentially checking whether the GPU has been requested, later on in the run() function it does some final checks with useful log outputs which were previously redundant due to having the is_available() here.